### DISCUSSION TITLE
auto-completion feature request: rank items that starts with __ lower

### DISCUSSION THREAD

**Discussion by quantum-booty** [helix-editor/helix#7775] [General]
<img width="300" alt="image" src="https://user-images.githubusercontent.com/28958399/227387185-3a081837-e342-4a5b-b56a-da0eab5a9b06.png">
Currently, items that start with __ are ranked higher than the rest. In Python, items that start with __ are generally protected variables or dunder methods, which should not be called directly. Therefore they should be ranked lower in auto-completion.

**Comment by pascalkuthe** [comment]
The intal order of  compeltuon items depends on the LSP, not helix so you should suggest this upstream
